tests/queries/tests.py

```diff
@@ -3792,3 +3794,47 @@ def test_ticket_23622(self):
             set(Ticket23605A.objects.filter(qy).values_list('pk', flat=True))
         )
         self.assertSequenceEqual(Ticket23605A.objects.filter(qx), [a2])
+
+
+class Ticket22550Tests(TestCase):
+    def setUp(self):
+        for model in SingleObject, SingleObjectWithoutOrdering:
+            model.objects.bulk_create(
+                model(name='name{}'.format(i)) for i in range(2)
+            )
+
+    def test_ticket_22550(self):
```

Comment:
This test is too long and is testing things unrelated to the validation that's added. Non-regression tests (that are already passing before this PR) should be added separately, if at all.